src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -335,7 +337,11 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
 
         {renderRoomVisibility()}
 
-        <Button type="submit" disabled={!dirty || isSubmitting}>
+        <Button
+          className="AdvancedSettings__save-button"
+          type="submit"
+          disabled={!dirty || isSubmitting}
+        >
```

Comment:
Maybe OOS, but if there is a fast few minute ButtonNG replacement possible - FTW

Reply:
Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3